daml-lf/transaction/src/main/scala/com/digitalasset/daml/lf/command.scala

```diff
@@ -30,4 +30,10 @@ package object command {
   @deprecated("use ApiCommands", since = "2.1.0")
   val Commands = ApiCommands
 
+  @deprecated("use com.daml.lf.transaction.DisclosedCreateEvent", since = "2.6.0")
+  type ProcessedDisclosedContract = transaction.DisclosedEvent
+
+  @deprecated("use com.daml.lf.transaction.DisclosedCreateEvent", since = "2.6.0")
```

Comment:
I don't find the reference. Maybe rename got in the way?

Reply:
Ah, I think it ended up as `DisclosedEvent` but I would actually use `DisclosedCreateEvent` like you suggested here. `DisclosedEvent` sounds generic even though it's about creates